gluten-substrait/src/main/scala/org/apache/gluten/config/GlutenConfig.scala

```diff
@@ -1583,6 +1587,13 @@ object GlutenConfig {
       .doubleConf
       .createWithDefault(2.0d)
 
+  val AUTO_ADJUST_STAGE_RESOURCES_OFFHEAP_RATIO =
+    buildConf("spark.gluten.auto.adjustStageResources.offheap.ratio")
+      .internal()
+      .doc("Experimental: Decrease executor offheap memory when match adjust stage resource rule.")
+      .doubleConf
+      .createWithDefault(1.0d)
```

Comment:
The default value means its size is not changed. Setting to 0.5 may be better.

Reply:
done